<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLSMainWindow</class>
 <widget class="QMainWindow" name="DLSMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>870</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="upperMainHorizontalLayout">
      <item>
       <widget class="QFrame" name="dupScanPanelFrame">
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="dupScanGroupBox">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>370</height>
            </size>
           </property>
           <property name="title">
            <string>DupScan Panel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QFrame" name="dupScanLevelFrame">
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>96</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <layout class="QHBoxLayout" name="dupScanLevelLayout">
               <item>
                <layout class="QVBoxLayout" name="dupScanVerticalLayout">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetFixedSize</enum>
                 </property>
                 <item>
                  <widget class="QRadioButton" name="stage1_setupRadioButton">
                   <property name="text">
                    <string>1. Setup DupScan</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="stage2_FindRadioButton">
                   <property name="text">
                    <string>2. Find Duplicates</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="stage3_ActionsRadioButton">
                   <property name="text">
                    <string>3. Actions</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="dupScanParameterFrame">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="outputMainVerticalLayout">
        <item>
         <widget class="QSplitter" name="output_and_msgInfoSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="dupScanFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>520</width>
             <height>280</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
          </widget>
          <widget class="QTextBrowser" name="msgInfoTextBrowser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
           <property name="openLinks">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="msgInfoTypeHLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Courier New</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>This is Not Rocket Science :-)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="msgInfoTypeHSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="showHideMsgInfoPushButton">
            <property name="text">
             <string>Show Log</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="msgInfoTypeComboBox">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="progressMainHLayout">
      <item>
       <layout class="QGridLayout" name="progressGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="progressBarFiniteLabel">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Progress: IDLE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="progressBarFinite">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>12</height>
           </size>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="progressBarWorkingLabel">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Working</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="progressBarWorking">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>11</height>
           </size>
          </property>
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="progressWidgetsHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="progressActionFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="progressSuspendPushButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>38</height>
            </size>
           </property>
           <property name="text">
            <string>SUSPEND</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="progressStopPushButton">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>38</height>
            </size>
           </property>
           <property name="text">
            <string>STOP</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="scanSpeedHSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="scanSpeedLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Scanning Speed:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="scanSpeedVerticalLayout">
           <item>
            <layout class="QHBoxLayout" name="scanSpeedTopSlowFastHLayout">
             <item>
              <widget class="QLabel" name="scanSpeedSlowLabel">
               <property name="font">
                <font>
                 <pointsize>6</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Slow</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="scanSpeedTopHSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="scanSpeedFastLabel">
               <property name="font">
                <font>
                 <pointsize>6</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Fast</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="scanSpeedHorizontalSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="pageStep">
              <number>20</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
             <property name="tickInterval">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="superSpeedCheckBox">
           <property name="text">
            <string>Super Speed</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_DupLichaSe_File"/>
    <addaction name="action_Save_DupLichaSe_File"/>
    <addaction name="separator"/>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Program">
    <property name="title">
     <string>&amp;Program</string>
    </property>
    <addaction name="action_DupLichaSe_DUPScan"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="separator"/>
    <addaction name="action_Report_A_Problem"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Program"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Open_DupLichaSe_File">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open DupLichaSe File</string>
   </property>
  </action>
  <action name="action_Save_DupLichaSe_File">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Save DupLichaSe File</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;NOT&lt;/span&gt; yet available . . . scheduled to be in&lt;/p&gt;&lt;p&gt;versions 1.6+ and above!!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_DupLichaSe_DUPScan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;DupLichaSe DUPScan</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;DupLichaSe DUPScan &lt;/span&gt;is currenttly the only module available to &lt;span style=&quot; font-weight:600;&quot;&gt;DupLichaSe&lt;/span&gt;.&lt;/p&gt;&lt;p&gt;From newer versions, other modules are expected to come in such as &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;+DupLichaSe DUPFuzzyScan&lt;/span&gt; - Data Fuzzy Scanner for consumer content like music, pictures and videos.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;+DupLichaSe DUPTagScan&lt;/span&gt; - Tag Fuzzy Scanner for various files with tags including multimedia files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_Report_A_Problem">
   <property name="text">
    <string>&amp;Report A Problem</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>DLSMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showHideMsgInfoPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>msgInfoTextBrowser</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
